scormxblock/scormxblock.py

```diff
@@ -309,10 +310,8 @@ def get_context_studio(self):
     def get_context_student(self):
         scorm_file_path = ''
         if self.scorm_file:
-            scheme = 'https' if settings.HTTPS == 'on' else 'http'
-            scorm_file_path = '{}://{}{}'.format(
-                scheme,
-                configuration_helpers.get_value('site_domain', settings.ENV_TOKENS.get('LMS_BASE')),
+            scorm_file_path = '{}{}'.format(
+                configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
```

Comment:
We don't set `LMS_ROOT_URL` in site_configuration, instead we override `LMS_ROOT_URL` via django settings override for all sites (https://edlyio.atlassian.net/wiki/spaces/BM/pages/713785351/Edly+Cloud+Client+Setup+Guide+on+Multisite+Instance). How about we update it like this:
```python
scorm_file_path = '{lms_root_url}{scorm_file_name}'.format(
    lms_root_url=settings.LMS_ROOT_URL,
    scorm_file_name=self.scorm_file,
)
```

Reply:
I think we should give preference to site config and fallback to settings if site config does have key. That is standard approach

Reply:
Got it. This change is for OpenedX community.